components/centraldashboard-angular/frontend/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve --proxy-config proxy.conf.json",
+    "start": "npm run build-library && ng serve --proxy-config proxy.conf.json",
     "build": "ng build",
```

Comment:
Should we change this to
```json
"build": "npm run build-library && ng build"
```
to ensure that the produced library ends up in the final docker image as well?

Reply:
My suggestion @kimwnasptd  is to add what you proposed on the comment above and since `ng build` already copies the built library into its assets folder (since we have the `glob`), then go ahead and add the following line in the `Dockerfile`, since that's where the `backend` expects to find the `dashboard_lib.bundle.js`.

```
  COPY --from=backend /src .
  COPY --from=frontend /src/dist/default/ ./dist/public/
+ COPY --from=frontend /src/dist/default/assets ./dist/public/

  EXPOSE 8082
  ENTRYPOINT ["npm", "start"]
```

Reply:
We should be aware that running `npm run build -- --output-path=./dist/default --configuration=production` in the Dockerfile only affects the output path of `ng build` and not `npm run build-library`.

Reply:
@orfeas-k maybe I'm missing something. Where in the current Dockerfile do we build the library and place the produced file under `dist/public`? 

I understand we have a bug now where you had built the library previously with `npm run build-library` and the docker build process ended up using that file, but want to confirm.

Reply:
> Where in the current Dockerfile do we build the library and place the produced file under `dist/public`?

@kimwnasptd Nowhere, that's why I think we should add the following line (that you proposed). `npm run build-library` will build the library and place the file under `dist/public`.
```
    "start": "npm run build-library && ng serve --proxy-config proxy.conf.json",
```

Reply:
Ah, OK I misinterpreted your first comment. Then my last question regarding adding this line:
```
  COPY --from=backend /src .
  COPY --from=frontend /src/dist/default/ ./dist/public/
+ COPY --from=frontend /src/dist/default/assets ./dist/public/
```
Do we need other files from `src/dist/default/assets` or just the built library? If it's just the library then let's explicitly copy that file

Reply:
Explicitly that file (and the corresponding  `.map`) so I just rebased and pushed a branch with a fixup for this one @kimwnasptd

Reply:
looks good, thanks!